near.go

```diff
@@ -0,0 +1,50 @@
+package osrm
+
+import geo "github.com/paulmach/go.geo"
+
+// NearRequest represents a request to the near method
+type NearRequest struct {
```

Comment:
Rename `Near*` to `Nearest*` everywhere. We use a convention to name types according to OSRM API.